proxy/src/serverless/sql_over_http.rs

```diff
@@ -287,6 +283,9 @@ pub async fn handle(
             )?
         }
     };
+
+    ctx.log();
```

Comment:
Hm, actually, why did you change it to explicit events reporting (instead of doing it within the destructor)?

Reply:
Yeah it's not necessary. left over from a prior change. Will remove